GoogleForCreators/web-stories-wp
# Redesign: Implemented size+position panel

## Context

Implements size&position panel as per #6468.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-03-07 at 14 44 25](https://user-images.githubusercontent.com/637548/110252553-0141c500-7f54-11eb-8a09-fc52ec51b330.png) | ![Screen Shot 2021-03-07 at 14 43 40](https://user-images.githubusercontent.com/637548/110252558-03a41f00-7f54-11eb-9b94-358de822fedc.png) |
| ![Screen Shot 2021-03-07 at 14 44 39](https://user-images.githubusercontent.com/637548/110252570-0d2d8700-7f54-11eb-8f39-55b8d4071758.png) | ![Screen Shot 2021-03-07 at 14 43 46](https://user-images.githubusercontent.com/637548/110252573-0f8fe100-7f54-11eb-8c21-c2766fc2b13b.png) |
| ![Screen Shot 2021-03-07 at 23 32 33](https://user-images.githubusercontent.com/637548/110298146-3edd3700-7fc2-11eb-927d-b2bacab9be78.png) | ![Screen Shot 2021-03-07 at 23 32 12](https://user-images.githubusercontent.com/637548/110298149-400e6400-7fc2-11eb-8964-abc49c4142a5.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Select a shape element
2. Manipulate it using the size+position panel
3. Observe that all functions still work, including flip toggles
4. Select another shape element as well at a different position and angle
5. Observe all text fields correctly displaying _"Mixed"_
6. Enter values in the mixed input fields to align the two shapes in position and/or size

### UAT




## Checklist




---

Fixes #6468
